src/domain/core/src/entities/dataset_pushes.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::collections::HashMap;
+
+use chrono::{DateTime, Utc};
+use opendatafabric::{DatasetRefRemote, Multihash};
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub struct DatasetPush {
+    pub target: DatasetRefRemote,
+    pub pushed_at: DateTime<Utc>,
```

Comment:
I couldn't find any uses of this field. Was it supposed to improve diagnostic messages?